.github/workflows/certora.yml

```diff
@@ -1,7 +1,5 @@
-# This is an example configuration file for running Certora verification
-# through github actions.  You can see the results for each push in the
-# "Actions" tab on the github website.
-#
+# A workflow file for running Certora verification through github actions.
+# Find results for each push in the "Actions" tab on the github website.
```

Comment:
Just emphasizing that this is a fully functional workflow file and a good starting point, ideally projects don't need to do much configuration within this file and it is largely the same for all projects

Reply:
Thanks @jflatow, i agree. i updated lines on master.